week6/sentimental-mario-more/mario.py

```diff
@@ -0,0 +1,12 @@
+while True:
+    try:
+        Height = int(input("Height: "))
+    except ValueError:
+        print("Height have to be an integer!")
+    if Height < 9 and Height > 0:
+        break
+    else:
+        print("Height have to be between 1 - 8")
+
+for i in range(Height):
+    print(" " * (Height - i), "#" * (i + 1), " ", "#" * (i + 1))
```

Comment:
Oneliners are cool, yeah.

It is however debatable wether a longer solution would increase readability. In this rather simple task i would say no.